presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -163,4 +166,33 @@ public GlueHiveMetastoreConfig setAwsCredentialsProvider(String awsCredentialsPr
         this.awsCredentialsProvider = Optional.ofNullable(awsCredentialsProvider);
         return this;
     }
+
+    @Min(1)
+    @Max(10)
+    public int getTotalSegments()
+    {
+        return totalSegments;
+    }
+
+    @Config("hive.metastore.glue.total-segments")
+    @ConfigDescription("Number of segments for partitioned Glue tables")
+    public GlueHiveMetastoreConfig setTotalSegments(int totalSegments)
+    {
+        this.totalSegments = totalSegments;
+        return this;
+    }
+
+    @Min(1)
+    public int getTotalGetPartitionThreads()
+    {
+        return totalGetPartitionThreads;
+    }
+
+    @Config("hive.metastore.glue.total-get-partition-threads")
+    @ConfigDescription("Number of threads for parallel partition fetches from Glue")
```

Comment:
@electrum how to say that 1 is a special value here?

Reply:
The difference is subtle and complicated to explain. I think the description here is fine, though we could document it in the main Hive documentation.